splunk-otel-android/src/main/java/com/splunk/rum/CurrentNetwork.java

```diff
@@ -16,19 +16,28 @@
 
 package com.splunk.rum;
 
+import android.os.Build;
 import androidx.annotation.Nullable;
 import java.util.Objects;
-import java.util.Optional;
 
 final class CurrentNetwork {
+    @Nullable private final Carrier carrier;
     private final NetworkState state;
     @Nullable private final String subType;
 
-    CurrentNetwork(NetworkState state, @Nullable String subType) {
+    private CurrentNetwork(
```

Comment:
Is this constructor used? It's private, doesn't seem to be referenced within this class

Reply:
Oops, I think it was left over after the builder was completed. I'll remove.